snap-overlay/templates/microstack.rc.j2

```diff
@@ -0,0 +1,9 @@
+export OS_PROJECT_DOMAIN_NAME=default
+export OS_USER_DOMAIN_NAME=default
+export OS_PROJECT_NAME=admin
+export OS_USERNAME=admin
+export OS_PASSWORD={{ ospassword }}
+export OS_AUTH_URL=http://{{ extgateway }}:5000
+export OS_IDENTITY_API_VERSION=3
+export OS_IMAGE_API_VERSION=2
+
```

Comment:
This is the source of the race condition that inspired the snap.openstack fork. Since we are writring and chmodding it so much, it's easy to hit it when it has been freshly written, but not yet made accessible.

Another solution would be to just use calls to snapctl rather than jinja variables, and move it out of the templating system. That makes it harder for people to grab and use outside of the snap environment, though, and doesn't fix the issue with writing and rewriting the templates ...